extra/mender-client-docker-addons/Dockerfile

```diff
@@ -6,8 +6,10 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y make git build-essential golang liblzma-dev \
     jq libssl-dev libglib2.0-dev curl
 
-ARG MENDER_CLIENT_REV=master
+# TODO - This cannot be hardcoded to anything else but master later
+ARG MENDER_CLIENT_REV=automatic_bootstrap_artifact
```

Comment:
Remember to fix this

Reply:
This we will have to fix after the whole pipeline is green.